zarr/_storage/store.py

```diff
@@ -254,6 +256,74 @@ def __setitem__(self, key, value):
     def __getitem__(self, key):
         """Get a value."""
 
+    @abc.abstractmethod
+    def rmdir(self, path=None):
+        """Remove a data path and all its subkeys and related metadata.
+        Expects a path without the data or meta root prefix."""
+
+    def supports_efficient_get_partial_values(self):
+        return False
+
+    def get_partial_values(self, key_ranges):
```

Comment:
A type hint would be very useful here.

Reply:
I used to have type-hints there in an early version, but removed them since the other methods didn't have some. Just out of curiosity: Is there some guideline when type-hints should be used? Or even a plan to add them everywhere?

I'll add them here where you commented :+1: